services/account-without-juggler/tslint.json

```diff
@@ -0,0 +1,72 @@
+{
+  // See https://palantir.github.io/tslint/rules/
```

Comment:
I think these comments were copied from loopback-next. I added this particular comment in loopback-next to make it easier for people that are not familiar with tslint to find a reference of all rules.

What are your objections against having these comments in place?